sdk/src/main/scala/com/horizen/SidechainWallet.scala

```diff
@@ -165,6 +168,40 @@ class SidechainWallet private[horizen] (seed: Array[Byte],
     this
   }
 
+  /***
+   * This function is called at blockchain bootstrap time and preload the SidechainWallletBoxStorage with the boxes taken from the backup storage
+   * @param backupStorageIterator: iterator on the backup storage
+   * @param sidechainBoxesCompanion
+   */
+  def scanBackUp(backupStorageIterator: StorageIterator, sidechainBoxesCompanion: SidechainBoxesCompanion): Unit = {
```

Comment:
The result type should be Try[SidecahinWallet] same as for other method-modifiers.

Reply:
Done